pkg/pillar/containerd/oci.go

```diff
@@ -113,16 +113,18 @@ func (s *ociSpec) AddLoader(volume string) error {
 	// massages .Mounts
 	if s.Root.Path != "/" { // FIXME-TASKS: it should be enough to give original OCI spec to a loader
 		volumeRoot := path.Join(s.Root.Path, "..")
-		// create full copy of our runtime spec
-		f, err := os.Create(filepath.Join(volumeRoot, ociRuntimeSpecFilename))
-		if err != nil {
-			return err
-		} else {
-			defer f.Close()
-		}
-		if err = s.Save(f); err != nil {
-			return err
-		}
+		/*
```

Comment:
One nit -- don't comment it out -- just remove it altogether (we can alway get it back from git history). But please DO leave a TODO comment in place explaining what it was trying to do and also explaining the circular nature we ran into when the code is there -- just so I don't forget to fix it properly once I'm back from vacation.

Makes sense @giggsoff ?

Reply:
Make sense. Will try to figure out for which purpose it is here for now.

Reply:
Done, please check